PCB-testing/chris-js/magnetometer-test.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * magnetometer-test.js
+ * Aims to test getting mangetometer data
+ * using I2C on the board designed by the
+ * electrical team.
+ *
+ * Author: Chris Thierauf <chris@cthierauf.com>
+ * A lot of stuff taken form Owen McAree here:
+ * https://github.com/omcaree/node-hmc6343
+ *
+ * Licensed under GPLv3, see details and license
+ * in root folder.
+ * Copyright (c) 2018 Nugget Industries
+ *
+ */
+
+// Include the module
+const hmc6343 = require('hmc6343.js');
+
+// Create new instance
+let compass = new hmc6343('/dev/i2c-3', 0x19);
```

Comment:
This can be `const`, the only difference between `const` and `let` is that a `let` can be re-assigned. a `const`'s JSON properties can still be changed an manipulated.

Reply:
Use `const` here if you catch my drift